docs/permissions.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Permissions
+---
+
+Administrate's base logic assumes any signed-in super-user
+can access any record and run any operation.
+You can manage permissions by placing logic in your controllers.
+
+[Pundit] is a popular gem many people describe permissions rules in.
+Administrate's Pundit plugin applies your permissions across your dashboards.
+See the end of this page.
```

Comment:
I think that mentioning Pundit from the get go will lead some users to think that it is a requirement.

Or perhaps this could be rephrased to clarify that this is not the case:

```suggestion
You can write this logic yourself using your preferred tools. Alternatively, Administrate
provides an easy path to integrate with [Pundit], a popular gem for describing permissions.
For more details, read on.
```